reference/normalize_test.go

```diff
@@ -133,28 +133,28 @@ func TestParseRepositoryInfo(t *testing.T) {
 	tcases := []tcase{
 		{
 			RemoteName:    "fooo/bar",
-			FamiliarName:  "fooo/bar",
+			FamiliarName:  "docker.io/fooo/bar",
 			FullName:      "docker.io/fooo/bar",
 			AmbiguousName: "index.docker.io/fooo/bar",
 			Domain:        "docker.io",
 		},
 		{
 			RemoteName:    "library/ubuntu",
-			FamiliarName:  "ubuntu",
+			FamiliarName:  "docker.io/library/ubuntu",
```

Comment:
I guess it's already too late to ask why library is kept as well. But may I ask for the reason behind removing it?

I find `docker.io/ubuntu` certainly more familiar.

Reply:
> I guess it's already too late to ask why library is kept as well. But may I ask for the reason behind removing it?

The reason was a result of this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1561989

Reply:
> I guess it's already too late to ask why library

That's a good question. Looking at this test again.

Reply:
> I guess it's already too late to ask why library is kept as well.

I think this test was originally asserting that if you parse `library/ubuntu` then calling `FamiliarName()` would not add back the repo component again (i.e., you don't end up with `docker.io/library/library/ubuntu`.